src/test/scala/uk/gov/hmrc/perftests/itsass/IncomeTaxSubmissionRequests.scala

```diff
@@ -29,11 +29,11 @@ object IncomeTaxSubmissionRequests extends ServicesConfiguration {
     .check(status.is(200))
 
   def getAgentTestOnlyEndPoint: HttpRequestBuilder = http("Get Agent Test Only End Point")
-    .get(s"$serviceUrl/test-only/2022/agent-access/1234567890")
+    .get(s"$serviceUrl/test-only/2022/additional-parameters?ClientNino=AA123456A&ClientMTDID=1234567890")
```

Comment:
Given this request is now the same as the one below, do we need it?

Reply:
I am unable to find  .get(s"$serviceUrl/test-only/2022/agent-access/1234567890")